MapleServer2/Managers/FieldManager.cs

```diff
@@ -1335,14 +1329,7 @@ private void HealingSpot()
                     continue;
                 }
 
-                int healAmount = (int) (player.Value.Stats[StatAttribute.Hp].Bonus * 0.03);
-                Status status = new(new(70000018, 1, 0, 1), player.ObjectId, healingSpot.ObjectId, 1);
-
-                player.Value.Session?.Send(BuffPacket.AddBuff(status));
-                BroadcastPacket(SkillDamagePacket.Heal(status, healAmount));
-
-                player.Stats[StatAttribute.Hp].AddValue(healAmount);
-                player.Value.Session?.Send(StatPacket.UpdateStats(player, StatAttribute.Hp));
+                player.AdditionalEffects.AddEffect(new(70000018, 1));
```

Comment:
What is this?

Reply:
Additional effects has been updated to support healing so I've been moving over anything that references an additional effect to use the system for it